src/service/NylasService.ts

```diff
@@ -137,6 +137,24 @@ export class NylasService extends APIClient {
             }
         });
     }
+
+    /**
+     * This method returns all existing Scheduler pages for the specified nylas account
+     * @param accountId
+     */
+    async getSchedulerPages(accountId: string): Promise<AxiosResponse> {
+        return await this.invokeApi(`/schedule/manage/pages?account_id=${accountId}`, 'GET');
```

Comment:
Wenn du beim invokeApi das Model mit angibst, bekommst du den korrekten Rückgabewert
```suggestion
        return await this.invokeApi<Model>(`/schedule/manage/pages?account_id=${accountId}`, 'GET');
```